Source/Client/Managers/ChatManager.cs

```diff
@@ -155,6 +154,61 @@ public static void ChatClock()
 
             chatClockTask = null;
         }
+
+        private static string ParseMessage(string msg)
+        {
+            string parsedMessage = msg;
+            string verification = "";
+            bool verifying = false;
+            Stack<string> codeType = new();
+
+            Regex colorRegex = new(@"\[[a-fA-F0-9]{6}\]");
+
+            foreach(char c in msg) 
+            {
+                if (c == '[') verifying = true;
+
+                if (verifying)
+                {
+                    verification += c;
+                    if (c == ']') verifying = false;
+                }
+                
+                if (verification != "" && !verifying)
+                {
+                    switch(verification)
+                    {
+                        case "[/]":
+                            if (codeType.Count > 0) parsedMessage = parsedMessage.ReplaceFirst(verification, $"</{codeType.Pop()}>");
+                            verification = "";
+                            break;
+                        case "[b]":
```

Comment:
This is nice formatting style but I would suggest changing to markdown as it is simpler for the user and does not use bracket notation.

Reply:
I'll do some tests later today, what about colors in markdown?
I used the bracket system to keep the same style for everything.

Reply:
you are right markdown does not have color, I forgot this fact.
My choice would be brackets for colors and markdown for formatting, but I am just a user.

Reply:
There, added a custom markdown for color codes: starts with `&rrggbb` and ends with `&&`